tests/test-http-api/test-config.js

```diff
@@ -221,5 +234,15 @@ describe('config', () => {
         })
       })
     })
+
+    describe('ipfs.config.show', () => {
```

Comment:
Was just trying to keep the structure of the other endpoints' tests, removed both of them